apps/src/redux/unitSelectionRedux.js

```diff
@@ -88,19 +87,6 @@ export default function unitSelection(state = initialState, action) {
     };
   }
 
-  // Note: This listens to the sectionData redux, not the unitSelection redux.
-  if (action.type === SET_SECTION) {
```

Comment:
I've moved setting the script ID to `teacher_dashboard/show.js` since section is only set on page load in the teacher dashboard.